apps/builder/app/dashboard/projects/project-card.tsx

```diff
@@ -180,11 +179,30 @@ export const ProjectCard = ({
   const [isHidden, setIsHidden] = useState(false);
   const { thumbnailRef, handleKeyDown } = useProjectCard();
   const handleCloneProject = useCloneProject(id);
-  const { state, location } = useNavigation();
+  const [isTransitioning, setIsTransitioning] = useState(false);
+
+  useEffect(() => {
+    const linkPath = builderUrl({ origin: window.origin, projectId: id });
+
+    const handleNavigate = (event: NavigateEvent) => {
+      if (event.destination.url === linkPath) {
+        setIsTransitioning(true);
+      }
+    };
+
+    if (window.navigation === undefined) {
```

Comment:
is it not available somewhere?

Reply:
my safari gives undefined

Reply:
will recheck

Reply:
<img width="378" alt="image" src="https://github.com/user-attachments/assets/9cc220bd-1cab-49ea-a2ee-265465b3f6b1">